connector/connect/docs/adding-proto-messages.md

```diff
@@ -0,0 +1,87 @@
+# Required, Optional and default values
+
+Connect adopts proto3, which does not support `required` constraint anymore.
+Every field is optional. For non-message proto fields, there is also no `hasXXX`
+functions to easy tell if a filed is set or not-set. For those non-message proto,
+they also have default values. E.g. int has 0 as the default value.
```

Comment:
Hey, please have a look at https://developers.google.com/protocol-buffers/docs/proto3#specifying_field_rules and check if this is not applicable in this case.